index.html

```diff
@@ -93,30 +106,38 @@ <h2><dfn>Serial</dfn> interface</h2>
 };
 </pre>
 
-<section>
-<h2><dfn>requestPort()</dfn> method</h2>
+The <dfn>onconnect</dfn> is an <a>EventHandler</a> which is called to notify
+when a connection is established.
```

Comment:
I think this wording is confusing. My intention for the `onconnect` and `ondisconnect` events is to notify script when the set of ports that would be returned by `getPorts()` has changed, not when a connection to a device attached to a port has been established or broken.

Reply:
We should probably fix this as a separate bug. Just also noting that this are "event handler attributes" per HTML's definition... not a big deal, but they have the magic behavior of being able to receive the events.

Reply:
I think we need better names then. Users will misunderstand what these events are for. Could it be a single `portschange` event?

Reply:
In the [WebUSB spec](https://wicg.github.io/webusb/#enumeration) we have an example which illustrates how to use `getDevices()` (`getPorts()` in this spec) and the `onconnect` and `ondisconnect` event handlers.

Reply:
Let’s move this to another bug, as otherwise we risk getting blocked on bikeshedding event names.